admin-dev/themes/new-theme/js/pages/product/components/filters/FilterDropdown.vue

```diff
@@ -91,9 +91,13 @@
         type: String,
         required: true,
       },
+      eventEmitter: {
+        type: Object,
+        required: true,
+      },
     },
     mounted() {
-      this.$parent.$on('clearAll', this.clear);
+      this.eventEmitter.on('clearAll', this.clear);
```

Comment:
Same comment as previously, should we really replace the "local" event with the global one? It seems that it changes the behavior and can potentially create some bugs because the events will be triggered throughout all the components

Reply:
$parent doesn't exist anymore, there is a workaround but not advised by the Vue team, avoiding this would require a bigger refactor that I would love to avoid there